go/cmd/vtctldclient/command/tablets.go

```diff
@@ -598,6 +630,7 @@ func init() {
 	Root.AddCommand(ExecuteHook)
 	Root.AddCommand(GetPermissions)
 	Root.AddCommand(GetTablet)
+	Root.AddCommand(GetFullStatus)
```

Comment:
annoying nitpick (i know, i know), can you keep this sorted? (should be above `GetTablet`)

Reply:
Fixed

Reply:
sorry, technically should be above `GetPermissions`.

Reply:
Fixed